entrypoints/content/pages/profile.tsx

```diff
@@ -0,0 +1,128 @@
+// Components:
+import { Avatar, AvatarFallback, AvatarImage } from '../components/ui/avatar'
+import { Button } from '../components/ui/button'
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from '../components/ui/dropdown-menu'
+import { Separator } from '../components/ui/separator'
+import Comment from '../components/secondary/comment/Comment'
+
+// Imports:
+import { Camera } from 'lucide-react'
+import { Link2Icon } from '@radix-ui/react-icons'
+
+// Typescript:
+import { TOPICS } from 'constants/database/comments-and-replies'
+import { FieldValue } from 'firebase-admin/firestore'
+
+// Functions:
+function Profile({ children }: { children: React.ReactNode }) {
+  // Constants:
+  const SAMPLE_USER_INFO = {
+    fullName: 'Ben Holmes',
+    username: '@BenHolmesDev',
+  }
+
+  const SAMPLE_CONTENT = `Right when I heard Stephen first talk I had a gut feeling that it was him. 
+
+Just the way he talked about the situation it's like he had already come to terms with something as crazy and as random as an explosion. 
+
+
+Bro had a eulogy ready for her and everything.`
+
+  const SAMPLE_COMMENT = {
```

Comment:
added fixtures just now, you can use them instead from `@/fixtures/comment` (check implementation)